Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -41,7 +41,7 @@ static PowerShellKernel()
             _writeStreamProperty = typeof(PSObject).GetProperty("WriteStream", BindingFlags.Instance | BindingFlags.NonPublic);
             Type writeStreamType = typeof(PSObject).Assembly.GetType("System.Management.Automation.WriteStreamType");
             _errorStreamValue = Enum.Parse(writeStreamType, "Error");
-            _outDefaultCommand = new CmdletInfo("Out-Default", typeof(OutDefaultCommand));
+            _outDefaultCommand = new CmdletInfo("Out-Default2", typeof(OutDefaultCommand));
```

Comment:
Is this change intended?

Reply:
Yes that's what he says in the description

Reply:
A comment explaining why would be nice though

Reply:
Comment added.